### ISSUE TITLE
Twitchy animations

### ISSUE THREAD

**Issue by CHADREX** [spider-themes/eazydocs#5]
Once again, please remove unnecessary code, especially from the bootstrap)

The solution to the problem, delete the line
\wp-content\plugins\eazydocs\assets\vendors\bootstrap\bootstrap.min.css

`@media (prefers-reduced-motion:no-preference){:root{scroll-behavior:smooth}}`

Or forcibly use the parameter `auto`

**Comment by mdjwel** [comment]
What issues are you facing for this code?

**Comment by CHADREX** [comment]
> What issues are you facing for this code?

It's about smooth scrolling after clicking on anchor links

**Comment by mdjwel** [comment]
Seems, there is no issue with the smooth scrolling. 
If I delete that code, the smooth scroll will not work. So we need to keep that code to enable smooth scrolling. It's a feature of the plugin.

Thanks

**Comment by CHADREX** [comment]
Right, smooth scrolling won't work if it's not enabled in the browser. I'm personally more comfortable with this option.

**Comment by mdjwel** [comment]
Yes, you need to enable smooth scrolling from the browser settings. If you disable it from the browser, it will not work. 
And we need that code using the browser's smooth scrolling feature. If we delete that code, you will not be able to use the smooth scrolling feature even with enabling it from the browser. 

Thanks

**Comment by CHADREX** [comment]
Um... Why would I want smooth scrolling if I forced it off in my browser (for example)...

Anyway, smooth scrolling breaks the smooth transition to anchor links
